<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>fermentableEditor</class>
 <widget class="QDialog" name="fermentableEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>661</width>
    <height>348</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Fermentable Editor</string>
  </property>
  <property name="configSection" stdset="0">
   <string notr="true">fermentableEditor</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QTabWidget" name="tabWidget_editor">
     <property name="tabPosition">
      <enum>QTabWidget::West</enum>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <property name="usesScrollButtons">
      <bool>false</bool>
     </property>
     <widget class="QWidget" name="tab_main">
      <attribute name="title">
       <string>Main</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <item row="2" column="4">
        <widget class="QCheckBox" name="checkBox_recommendMash">
         <property name="toolTip">
          <string>Recommend this be mashed</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_name">
         <property name="toolTip">
          <string>Required</string>
         </property>
         <property name="text">
          <string>Name</string>
         </property>
         <property name="buddy">
          <cstring>lineEdit_name</cstring>
         </property>
        </widget>
       </item>
       <item row="8" column="1">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="2">
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="2" column="1">
        <widget class="QComboBox" name="comboBox_fermentableType">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="toolTip">
          <string>Fermantable Type</string>
         </property>
        </widget>
       </item>
       <item row="2" column="3">
        <widget class="QLabel" name="label_recommendMash">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Recommend Mash</string>
         </property>
         <property name="buddy">
          <cstring>checkBox_recommendMash</cstring>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QLabel" name="label_addAfterBoil">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Add After Boil</string>
         </property>
         <property name="buddy">
          <cstring>checkBox_addAfterBoil</cstring>
         </property>
        </widget>
       </item>
       <item row="0" column="4">
        <widget class="QCheckBox" name="checkBox_addAfterBoil">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="toolTip">
          <string>This ingredient is added post boil.</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="SmartLineEdit" name="lineEdit_name">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="toolTip">
          <string>Name</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_2">
         <property name="toolTip">
          <string>Required</string>
         </property>
         <property name="text">
          <string>Type</string>
         </property>
         <property name="buddy">
          <cstring>comboBox_fermentableType</cstring>
         </property>
        </widget>
       </item>
       <item row="3" column="3">
        <widget class="QLabel" name="label_isMashed">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Is Mashed</string>
         </property>
         <property name="buddy">
          <cstring>checkBox_isMashed</cstring>
         </property>
        </widget>
       </item>
       <item row="3" column="4">
        <widget class="QCheckBox" name="checkBox_isMashed">
         <property name="toolTip">
          <string>Checked if it is present in mash</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="7" column="0">
        <widget class="SmartLabel" name="label_inventory">
         <property name="contextMenuPolicy">
          <enum>Qt::CustomContextMenu</enum>
         </property>
         <property name="text">
          <string>Amount in Inventory</string>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
         <property name="buddy">
          <cstring>lineEdit_inventory</cstring>
         </property>
        </widget>
       </item>
       <item row="7" column="1">
        <widget class="SmartLineEdit" name="lineEdit_inventory">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="toolTip">
          <string>Amount in inventory</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="SmartLabel" name="label_color">
         <property name="contextMenuPolicy">
          <enum>Qt::CustomContextMenu</enum>
         </property>
         <property name="toolTip">
          <string>Required</string>
         </property>
         <property name="text">
          <string>Color</string>
         </property>
         <property name="buddy">
          <cstring>lineEdit_color</cstring>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="SmartLineEdit" name="lineEdit_color">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="toolTip">
          <string>Lovibond rating</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="label_yield">
         <property name="toolTip">
          <string>Required</string>
         </property>
         <property name="text">
          <string>Yield %</string>
         </property>
         <property name="buddy">
          <cstring>lineEdit_yield</cstring>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="SmartLineEdit" name="lineEdit_yield">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="toolTip">
          <string>Yield as compared to glucose</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_extras">
      <attribute name="title">
       <string>Extras</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="2" column="1">
        <widget class="SmartLineEdit" name="lineEdit_moisture">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="toolTip">
          <string>Moisture percentage by mass</string>
         </property>
        </widget>
       </item>
       <item row="3" column="3">
        <widget class="QLabel" name="label_ibuGalPerLb">
         <property name="text">
          <string>Bitterness (IBU*gal/lb)</string>
         </property>
         <property name="buddy">
          <cstring>lineEdit_ibuGalPerLb</cstring>
         </property>
        </widget>
       </item>
       <item row="3" column="4">
        <widget class="SmartLineEdit" name="lineEdit_ibuGalPerLb">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="toolTip">
          <string>Bitterness of pre-hopped extracts</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_moisture">
         <property name="text">
          <string>Moisture (%)</string>
         </property>
         <property name="buddy">
          <cstring>lineEdit_moisture</cstring>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="SmartLineEdit" name="lineEdit_origin">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="toolTip">
          <string>Origin</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_origin">
         <property name="text">
          <string>Origin</string>
         </property>
         <property name="buddy">
          <cstring>lineEdit_origin</cstring>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QLabel" name="label_protein">
         <property name="text">
          <string>Protein (%)</string>
         </property>
         <property name="buddy">
          <cstring>lineEdit_protein</cstring>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="SmartLabel" name="label_diastaticPower">
         <property name="contextMenuPolicy">
          <enum>Qt::CustomContextMenu</enum>
         </property>
         <property name="text">
          <string>Diastatic power</string>
         </property>
         <property name="buddy">
          <cstring>lineEdit_diastaticPower</cstring>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="SmartLineEdit" name="lineEdit_diastaticPower">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="toolTip">
          <string>Diastatic power</string>
         </property>
        </widget>
       </item>
       <item row="2" column="3">
        <widget class="QLabel" name="label_coarseFineDiff">
         <property name="text">
          <string>Coarse/Fine Diff (%)</string>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
         <property name="buddy">
          <cstring>lineEdit_coarseFineDiff</cstring>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_supplier">
         <property name="text">
          <string>Supplier</string>
         </property>
         <property name="buddy">
          <cstring>lineEdit_supplier</cstring>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="SmartLineEdit" name="lineEdit_supplier">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="toolTip">
          <string>Supplier</string>
         </property>
        </widget>
       </item>
       <item row="1" column="4">
        <widget class="SmartLineEdit" name="lineEdit_maxInBatch">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="toolTip">
          <string>Maximum recommended percentage of total grist</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="4">
        <widget class="SmartLineEdit" name="lineEdit_protein">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="toolTip">
          <string>Protein percentage by mass</string>
         </property>
        </widget>
       </item>
       <item row="1" column="3">
        <widget class="QLabel" name="label_maxInBatch">
         <property name="text">
          <string>Max In Batch (%)</string>
         </property>
         <property name="buddy">
          <cstring>lineEdit_maxInBatch</cstring>
         </property>
        </widget>
       </item>
       <item row="2" column="4">
        <widget class="SmartLineEdit" name="lineEdit_coarseFineDiff">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="toolTip">
          <string>Yield difference between coarse and fine grind</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_notes">
      <attribute name="title">
       <string>Notes</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QTextEdit" name="textEdit_notes">
         <property name="maximumSize">
          <size>
           <width>1000</width>
           <height>1000</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_new">
       <property name="toolTip">
        <string>New fermentable</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../brewken.qrc">
         <normaloff>:/images/smallPlus.svg</normaloff>:/images/smallPlus.svg</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_save">
       <property name="toolTip">
        <string>Save and close</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../brewken.qrc">
         <normaloff>:/images/filesave.svg</normaloff>:/images/filesave.svg</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_cancel">
       <property name="toolTip">
        <string>Discard and close</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../brewken.qrc">
         <normaloff>:/images/exit.svg</normaloff>:/images/exit.svg</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SmartLabel</class>
   <extends>QLabel</extends>
   <header>widgets/SmartLabel.h</header>
   <slots>
    <signal>changedSystemOfMeasurementOrScale(PreviousScaleInfo)</signal>
    <slot>popContextMenu(QPoint)</slot>
   </slots>
  </customwidget>
  <customwidget>
   <class>SmartLineEdit</class>
   <extends>QLineEdit</extends>
   <header>widgets/SmartLineEdit.h</header>
   <slots>
    <signal>textModified()</signal>
    <slot>lineChanged()</slot>
    <slot>lineChanged(PreviousScaleInfo)</slot>
   </slots>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>tabWidget_editor</tabstop>
  <tabstop>lineEdit_name</tabstop>
  <tabstop>comboBox_fermentableType</tabstop>
  <tabstop>lineEdit_color</tabstop>
  <tabstop>lineEdit_yield</tabstop>
  <tabstop>lineEdit_inventory</tabstop>
  <tabstop>checkBox_addAfterBoil</tabstop>
  <tabstop>checkBox_recommendMash</tabstop>
  <tabstop>checkBox_isMashed</tabstop>
  <tabstop>lineEdit_origin</tabstop>
  <tabstop>lineEdit_supplier</tabstop>
  <tabstop>lineEdit_moisture</tabstop>
  <tabstop>lineEdit_diastaticPower</tabstop>
  <tabstop>lineEdit_protein</tabstop>
  <tabstop>lineEdit_maxInBatch</tabstop>
  <tabstop>lineEdit_coarseFineDiff</tabstop>
  <tabstop>lineEdit_ibuGalPerLb</tabstop>
  <tabstop>pushButton_new</tabstop>
  <tabstop>pushButton_save</tabstop>
  <tabstop>pushButton_cancel</tabstop>
  <tabstop>textEdit_notes</tabstop>
 </tabstops>
 <resources>
  <include location="../brewken.qrc"/>
 </resources>
</ui>
